src/main/java/org/folio/rest/impl/LostItemFeePoliciesAPI.java

```diff
@@ -94,7 +103,28 @@ public void putLostItemFeesPoliciesByLostItemFeePolicyId(String lostItemFeePolic
     Handler<AsyncResult<Response>> asyncResultHandler,
     Context vertxContext) {
 
+    if (shouldRefuseWhenLostItemProcessingFeeNegative(entity)) {
+      refuseWhenLostItemProcessingFeeNegative(entity, asyncResultHandler);
+      return;
+    }
+
     PgUtil.put(TABLE_NAME, entity, lostItemFeePolicyId, okapiHeaders, vertxContext,
       PutLostItemFeesPoliciesByLostItemFeePolicyIdResponse.class, asyncResultHandler);
   }
+
+  private void refuseWhenLostItemProcessingFeeNegative(LostItemFeePolicy entity,
+    Handler<AsyncResult<Response>> asyncResultHandler) {
+
+      asyncResultHandler.handle(
+        succeededFuture(respond422WithApplicationJson(createError(
+          NEGATIVE_VALUE_MESSAGE, "lostItemProcessingFee",
+          entity.getLostItemProcessingFee().getAmount().toString()))));
+  }
+
+  private boolean shouldRefuseWhenLostItemProcessingFeeNegative(LostItemFeePolicy entity) {
+    return Optional.ofNullable(entity)
+      .map(LostItemFeePolicy::getLostItemProcessingFee)
+      .map(MonetaryValue::isNegative)
+      .orElse(false);

```

Comment:
Make 1 method that returns boolean and handles the error

Reply:
The methods are coupled, thanks.